example/rpi_led.py

```diff
@@ -23,34 +24,24 @@
 anto = antolib.Anto(user, key, thing)
 
 def connectedCB():
-    anto.sub("led1");
-    anto.sub("led2");
-    anto.sub("led3");
+    anto.sub("led1")
+    anto.sub("led2")
+    anto.sub("led3")
 
 def dataCB(channel, msg):
-    if(channel == 'led1'):
-        value = int(msg)
-        GPIO.output(leds[0], value)
-        if(value == 1):
-            print('led1: ON\n')
-        else:
-            print('led1: OFF\n')
+    value = int(msg)
 
-    elif(channel == 'led2'):
-        value = int(msg)
+    if channel == 'led1':
+        GPIO.output(leds[0], value)
+    elif channel == 'led2':
         GPIO.output(leds[1], value)
-        if(value == 1):
-            print('led2: ON\n')
-        else:
-            print('led2: OFF\n')
-
-    elif(channel == 'led3'):
-        value = int(msg)
+    elif channel == 'led3':
         GPIO.output(leds[2], value)
-        if(value == 1):
-            print('led3: ON\n')
-        else:
-            print('led3: OFF\n')
+
+    if value == 1:
```

Comment:
factorized to the right because it's almost the same operation for all cases, just adapted the code to work as expected